core/src/main/java/io/kestra/plugin/core/debug/Wait.java

```diff
@@ -0,0 +1,55 @@
+package io.kestra.plugin.core.debug;
```

Comment:
The package is not the good one, here is the one asked in the original issue : 
`io.kestra.plugin.core.flow.Sleep`
with `io.kestra.plugin.core.flow` being the package and `Sleep` being the class name
and this will form the task type

Reply:
@yoyounik Please read this, better take your times to make correct changes once than doing it 10 times

Reply:
Hi @Skraye 
Thanks for the feedback. You're right, "Sleep" might be a more descriptive name for the class.

I can rename the Wait.java class to Sleep.java and update the code accordingly. This would keep the file in the io.kestra.plugin.core.flow package.

Does that sound like a good approach?

Reply:
This is exactly what's need to be done :+1:

Reply:
okay then i will rename it and update you @Skraye

Reply:
Hey @Skraye 
changes i made:
Wait.java changed to Sleep.java,
wait-task-flow.yml changed to sleep-task-flow.yml and changes inside it to Sleep.
sleep.yml's  type has been changed.

going to update these changes now, please go through this once, and thanks for the patience shown :).
Please update me if any changes required.